news/2018-05-14-twc-finals-recap.md

```diff
@@ -0,0 +1,53 @@
+
+When Canada went up against Taiwan, their memorization game was on point as they opened the match with a win on the SV map [cranky - R176](https://osu.ppy.sh/beatmaps/1623411). After that, Taiwan remembered their warmup map [Dave Rodgers - Deja Vu](https://osu.ppy.sh/beatmaps/1637925). **They suddenly remembered that in order to keep up with tradition, they'd have to make it to the Grand Finals.** In order to get there, they hit full throttle to the point where Canada could not keep up. Taiwan had secured their Grand Finals spot over Canada 7-1.
+
+As for the Grand Finals, Taiwan was still going at full speed since the last match as they won the first 3 points against Japan. If that in itself isn't a historical achievement, Japan had to suffer 2 more maps lost as they clawed their way back to a 7-5 victory. Taiwan truly played their hearts out and made the Grand Final one of the most even matches of the tournament. Taiwan may have lost the match, but atleast they ended the tournament with a win on the tiebreaker showcase: 
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/japan-taiwan-tb.jpg)
+
+And that wraps up all the matches for this year! Here's how the entire elimination bracket went down:
+
+![](/wiki/shared/news/2018-05-07-twc-finals-recap/bracket.png)
+
+
+### Finals Mappool
+
+There was bound to be diversity in the picks, since teams required a whole 7 map wins in order to win in this stage. As the work for the mappool selectors this year has come to an end, here are the closing comments of one of them:
+
+[mangomizer](https://osu.ppy.sh/users/1893718):
+
+"It may come as a surprise to you that the Finals Mappool was the second mappool we started work on, immediately following Group Stage. This was done as a precaution to avoid having a repeat of the thermal runaway mappool that almost killed everyone last year. In spite of this, the Finals Mappool still ended up being extremely difficult – perhaps even more challenging than last year. We are thankful to have started this early, as it gave us more time to ensure the balance of the mappool.
+
+With regards to the matches last weekend, I’m certainly glad to see a fairly even spread of picks across the board. [“Nanoka - Tsukimade Todoke, Fushi no Kemuri”](https://osu.ppy.sh/beatmapsets/763774/#taiko/1605963) was probably my greatest oversight – I had seriously underestimated the difficulty it would pose on the players. Otherwise, I am seriously impressed with how players are coping with this mappool – their stellar performances exceeding my expectations more times than I can count. It has been an absolute pleasure to mappool select this year, and I cannot wait to do it again next year.”
+
+### Conclusion
+
+We saw some mid-seeded teams really step up this year. Hong Kong taking Canada to tiebreaker, Spain reaching top 8 and South Korea reaching top 6 are all signs of the skill gap between teams evening out, so let's hope this trend continues in future cups!
```

Comment:
```
We saw some mid-seeded teams really step up this year. Hong Kong tied with Canada, Spain reached top 8, and South Korea reached top 6. These are all signs of the skill gap between teams evening out, so let's hope this trend continues in future cups!
```

If you do use that, then maybe use `These appear to be`  instead of `These are` to make it sound more like an observation than a statement?